HuntBuddy/Windows/LocalHuntsWindow.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System.Linq;
+using System.Numerics;
+using Dalamud.Interface;
+using Dalamud.Interface.Windowing;
+using HuntBuddy.Utils;
+using ImGuiNET;
+
+namespace HuntBuddy.Windows;
+
+/// <summary>
+/// Local hunts window.
+/// </summary>
+public class LocalHuntsWindow : Window
+{
+    public LocalHuntsWindow() : base(
+        $"{Plugin.Instance.Name}",
+        ImGuiWindowFlags.NoNavInputs | ImGuiWindowFlags.NoDocking,
+        true)
+    {
+        this.Size = Vector2.Zero;
+        this.SizeCondition = ImGuiCond.Always;
+
+        if (Plugin.Instance.Configuration.HideLocalHuntBackground)
```

Comment:
This also needs to go in the pre-draw function, same issue as the config window

Reply:
Addressed in 93872f1